### DISCUSSION TITLE
How to get avatars for roster list after reinstall app?

### DISCUSSION THREAD

**Discussion by Vladimir7i** [processone/ejabberd#3912] [Questions]
I use vCard + xep-0084 for work with Avatars,
after reinstall application how I can take avatars for each friend in rosterList? (it's can be around 200 users)

take vCard for each friend - it's ok? or what the best way?

**Comment by licaon-kter** [comment]
You write your own application?

**Reply by Vladimir7i** [reply]
yeah(for game friendlist - connect from unity), if I put avatar(16x16 pix) in Nick(string) on Roster - it's good way? other words: Nick|avatarBase64 = and split it on client.
RosterList cached on client.

or I just can use to get vCard for that?
